docs/getting-started-docker-compose.md

```diff
@@ -0,0 +1,119 @@
+---
+id: getting-started-docker-compose
+title: Run a Pulsar cluster locally with `docker-compose`
+sidebar_label: "Run Pulsar locally with `docker-compose`"
```

Comment:
```suggestion
title: Run a Pulsar cluster locally with Docker Compose
sidebar_label: "Run Pulsar locally with Docker Compose"
```

Use the formal text name. See https://docs.docker.com/compose/.

Reply:
Thanks for pointing it out. Resolved in [b52acfa](https://github.com/apache/pulsar-site/pull/448/commits/b52acfaad99aefe158556af2bb4c442fa4ca5809).